tests/features/environment.py

```diff
@@ -18,8 +21,13 @@ def before_scenario(context, scenario):
     context.blaise_service = app.blaise_service
     context.totalmobile_service = app.totalmobile_service
     context.datastore_service = FakeDatastoreService()
+
     context.questionnaire_service = QuestionnaireService(
-        app.blaise_service, EligibleCaseService(), context.datastore_service
+        app.blaise_service,
+        EligibleCaseService(
```

Comment:
That factory again ;)

Reply:
Part of protected learning session I will run with James